test/test_ensemble.py

```diff
@@ -0,0 +1,20 @@
+from test.models.std_normal import StdNormal
+from bayes_kit.ensemble import AffineInvariantWalker
+import numpy as np
+
+def test_aiw_std_normal() -> None:
+    # init with draw from posterior
+    init = np.random.normal(loc=0, scale=1, size=[1])
+    model = StdNormal()
+    sampler = AffineInvariantWalker(model, a = 2, walkers=10)
+    M = 10
+    for m in range(M):
+        theta = sampler.sample()
+        print(theta)
+    return 1
```

Comment:
This looks like it was left in by accident, and I think it is preventing the test from actually testing anything